o2.sh

```diff
@@ -124,10 +127,11 @@ cmake $SOURCEDIR -DCMAKE_INSTALL_PREFIX=$INSTALLROOT
       ${ARROW_ROOT:+-DARROW_HOME=$ARROW_ROOT}                                               \
       -Dbenchmark_DIR=${GOOGLEBENCHMARK_ROOT}/lib/cmake/benchmark
 
-if [[ $GIT_TAG == master ]]; then
+if [[ $GIT_TAG == master && "${CMAKE_GENERATOR}" == "Unix Makefiles" ]]; then
```

Comment:
OK so I'd just drop entirely the whole `CONTINUE_ON_ERROR` logic while you're at it. It's nonsense to me (and we don't use it).

Reply:
ok